### ISSUE TITLE
Meilisearch Plugin

### ISSUE THREAD

**Issue by doreilly257** [backstage/backstage#8267]
Allow the use of Meilisearch indexes across content.

https://github.com/meilisearch/MeiliSearch

Same as other search plugins, build index from content and allow searches across those indexes.

https://github.com/meilisearch/MeiliSearch
https://docs.meilisearch.com/

**Comment by Rugvip** [comment]
Thank you for the suggestion! Just for clarity, the suggestion is to add this as a possible search plugin module right? Similar to https://github.com/backstage/backstage/tree/master/plugins/search-backend-module-elasticsearch

**Comment by doron050** [comment]
Hi @Rugvip
@SockworkOrange and I have looked at this issue, and want to give it a shot.

We don't have an implementation plan yet, but the working process will be to add a search plugin module like https://github.com/backstage/backstage/tree/master/plugins/search-backend-module-elasticsearch, then review the https://github.com/meilisearch/MeiliSearch docs and installation guide to implement the new plugin, something like `search-backend-module-meilisearch`

</br>

Does this sound acceptable? if so, you can assign us to the ticket 🙏
Thanks

**Comment by doron050** [comment]
bumping again, any update? @Rugvip

**Comment by Rugvip** [comment]
@doron050 yep plan for a `search-backend-module-meilisearch` sounds good 👍

**Comment by awanlin** [comment]
@Rugvip, should this be in https://github.com/backstage/community-plugins though?

**Comment by freben** [comment]
@awanlin eventually yes, but for now search and the other "core" plugins _and_ their modules live in the main repo.

**Comment by doron050** [comment]
Hi, I am updating you that, due to personal reasons, we don't have the capacity to take this on ourselves (@SockworkOrange and I)
The ticket is open for grabs if anyone wishes it.
